.github/CODEOWNERS

```diff
@@ -7,31 +7,16 @@
 # be requested for review when someone opens a pull request.
 # order is alphabetical for easier maintenance.
 #
-# Sean Gallacher (gallacher)
-# Trevor Dawe (tdawe)
-# Alexander Hoppe (hoppea2)
-# Alik Saring (alikdell)
 # Aaron Tye (atye)
-# Florian Coulombel (coulof)
-# Shayna Finocchiaro (shaynafinocchiaro)
-# Sharmila Ramamoorthy (sharmilarama)
+# Aly Nathoo (anathoodell)
+# Chiman Jain (chimanjain)
 # Evgeny Uglov (EvgenyUglov)
-# Bowen Jiang (bjiang27)
-# Tao He (taohe1012)
-# Peter Cao (P-Cao)
-# Yiming Bao (baoy1)
-# Yian Zong  (YianZong)
-# Forrest Xia (forrestxia)
-# Don Khan (donatwork)
-# Harsha Yalamanchili (harshaatdell)
-# AnandatDell (anandrajak1)
+# Harish H  (HarishH-DELL)
 # Nitesh Rewatkar (nitesh3108)
+# Oleksandr babiychuk (alexemc)
```

Comment:
Last name should start with a capital letter.

Reply:
Fixed.